src/main/java/edu/kit/compiler/JavaEasyCompiler.java

```diff
@@ -3,48 +3,80 @@
 import lombok.extern.slf4j.Slf4j;
 import org.apache.commons.cli.*;
 
-import java.io.File;
+import java.io.BufferedReader;
 import java.io.FileInputStream;
 import java.io.IOException;
-import java.util.Locale;
+import java.io.InputStreamReader;
 
 @Slf4j
 public class JavaEasyCompiler {
 
-    public static String byteArrayToHex(byte[] a) {
-        StringBuilder sb = new StringBuilder(a.length * 2);
-        for (byte b : a)
-            sb.append(String.format("%02x", b));
-        return sb.toString().toUpperCase(Locale.ROOT);
+    public static Result main(CommandLine cmd, Options options) {
```

Comment:
Braucht das wirklich zwei separate Funktionen? Würde einfach in main zuerst den try-Block fürs Parsen abhandeln und dann die verschiedenen Aufrufe im Anschluss. Wenn du zwei Funktionen behalten willst, bitte zumindest umbenennen (executeCommand?).

Reply:
Hm, andererseits, wenn wir für alle Commands eigene Funktionen machen (hätte ich vorgeschlagen), könnten wir es auch in eine Funktion ziehe.

Meinungen dazu?